WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Load WordPress/MigrationLoadWordPressViewController.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+class MigrationLoadWordPressViewController: UIViewController {
+
+    // MARK: - Dependencies
+
+    private let viewModel: MigrationLoadWordPressViewModel
+
+    // MARK: - Init
+
+    init(viewModel: MigrationLoadWordPressViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - View Lifecycle
+
+    override func loadView() {
+        let migrationView = MigrationStepView(
+            headerView: MigrationHeaderView(configuration: viewModel.header),
+            actionsView: MigrationActionsView(configuration: viewModel.actions),
+            centerView: UIView()
+        )
+        migrationView.additionalContentInset.top = 30
```

Comment:
@dvdchr @twstokes If this view controller doesn't have a navigation bar, I think we can remove this line to keep the UI consistent with other screens that don't show the navigation bar. The default value of `additionalContentInset.top` is `UINavigationBar().intrinsicContentSize.height` ( 44pt ).

Reply:
Cool! Completed in 4efcab0d910da913ffcec8fc60c99cb80ae73522.